apps/breakout-game/src/contexts/GameContext.tsx

```
@@ -1,12 +1,13 @@
 "use client"
 
-import { useEffect, useMemo } from "react"
+import { useEffect, useMemo, useContext } from "react"
 
 import type React from "react"
 
 import { createContext, useContext, useState, useRef, type ReactNode } from "react"
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useContext'. Consider to delete it or rename it.

'useContext' is defined here:


(lint/suspicious/noRedeclare)